src/test/java/org/jabref/model/BibDatabaseContextTest.java

```diff
@@ -42,10 +43,10 @@ public void testTypeBasedOnInferredModeBibTeX() {
     @Test
     public void testTypeBasedOnInferredModeBiblatex() {
         BibDatabase db = new BibDatabase();
-        BibEntry e1 = new BibEntry("electronic");
+        BibEntry e1 = new BibEntry(BibtexEntryTypes.ARTICLE);
         db.insertEntry(e1);
 
         BibDatabaseContext bibDatabaseContext = new BibDatabaseContext(db);
-        assertEquals(BibDatabaseMode.BIBLATEX, bibDatabaseContext.getMode());
+        assertEquals(BibDatabaseMode.BIBTEX, bibDatabaseContext.getMode());
```

Comment:
Here you change the logic of the test. 
Can you please use 'BiblatexEntryTypes.ELECTRONIC' in line 46 and keep the BibDatabaseMode.BIBLATEX here?

Reply:
Whoops. good catch!

Reply:
Sure